WikiCohereEnglishEmbeddingOnlyStreamingScenario.cs

```diff
@@ -1,17 +1,20 @@
-﻿using Microsoft.Extensions.Configuration;
+﻿using Microsoft.Azure.Cosmos;
+using Microsoft.Extensions.Configuration;
 
 namespace VectorIndexScenarioSuite
 {
-    internal class WikiCohereEnglishEmbeddingOnlyStreamingScenario : WikiCohereEmbeddingOnlyBaseSceario
+    internal class WikiCohereEnglishEmbeddingOnlyStreamingScenario : WikiCohereEnglishEmbeddingOnlyScenario
     {
+       
+        protected override string RunName => "wiki-cohere-english-embedding-only-streaming-" + guid;
         private const string RUNBOOK_PATH = "runbooks/wikipedia-35M_expirationtime_runbook.yaml";
         private const string GROUND_TRUTH_FILE_PREFIX_FOR_STEP = "step";
 
         /* This dataset comes with ground truth computed till 100 Nearest Neighbors. */
         private const string GROUND_TRUTH_FILE_EXTENSION_FOR_STEP = ".gt100";
 
         public WikiCohereEnglishEmbeddingOnlyStreamingScenario(IConfiguration configurations) : 
-            base(configurations, ComputeInitialAndFinalThroughput(configurations).Item1)
+            base(configurations)
         { }
 
         public override void Setup()

```

Comment:
For completeness, allow override in this scenarion as well if config 'cosmosContainerRUInitial' / 'cosmosContainerRUFinal' are populated.